src/server/scripts/Northrend/Ulduar/Ulduar/boss_xt002.cpp

```diff
@@ -374,7 +376,7 @@ class boss_xt002 : public CreatureScript
                     heart->CastSpell(heart, SPELL_HEART_OVERLOAD, false);
                     heart->CastSpell(me, SPELL_HEART_LIGHTNING_TETHER, false);
                     heart->CastSpell(heart, SPELL_HEART_HEAL_TO_FULL, true);
-                    heart->CastSpell(heart, SPELL_EXPOSED_HEART, false);    // Channeled
+                    heart->AddAura(SPELL_EXPOSED_HEART, heart);    // Changed 6-15-16, Hack Fix that applies 2x damage aura
```

Comment:
git blame already shows when the line was changed, you can remove the "Changed 6-15-16" part